README.md

```diff
@@ -28,6 +28,7 @@ Setup the application:
 
     $ cp development.ini.sample development.ini
     $ gearbox setup-app
+`If gearbox setup-app fails try specifying pip3 in the last two steps`
```

Comment:
Please only use "`"s to denote code blocks:

    If `gearbox setup-app` fails, try specifying `pip3` in the last two steps.

Also, note the comma after "fails".